### DISCUSSION TITLE
goguardian disabler for 106 and up

### DISCUSSION THREAD

**Discussion by GlizxyBoi** [3kh0/ext-remover#441] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there one for ext disabler for v 106 and up???

**Comment by Aka-but-nice** [comment, marked as answer]
Well, ever since GoGuardian deleted that one XSS file, we have not been able to find another. You could try old Point Blank, or Killcurly.

**Reply by GlizxyBoi** [reply]
> Well, ever since GoGuardian deleted that one XSS file, we have not been able to find another. You could try old Point Blank, or Killcurly.

hmm what was the XSS file again?

**Reply by Aka-but-nice** [reply]
This was [it](https://tinyurl.com/goofguardian)

**Comment by rwg100** [comment]
use killcurly, its literally the best and disabled all of the extensions including other ones that are filtering.

**Reply by GlizxyBoi** [reply]
> use killcurly, its literally the best and disabled all of the extensions including other ones that are filtering.

dont work

**Reply by Aka-but-nice** [reply]
I doesn't work for goguardian

**Reply by Aka-but-nice** [reply]
> It don't work for GoGuardian

**Reply by GlizxyBoi** [reply]
> > It don't work for GoGuardian

o
